src/lib/utils/preconditions/IPreconditionContainer.ts

```diff
@@ -33,5 +34,5 @@ export interface IPreconditionContainer {
 	 * @param message The message that ran this precondition.
 	 * @param command The command the message invoked.
 	 */
-	run(message: Message, command: Command): PreconditionContainerReturn;
+	run(message: Message, command: Command, context: PreconditionContext): PreconditionContainerReturn;
```

Comment:
Isn't this added required parameter a breaking change? Please document it if so.